web/src/message_edit.ts

```diff
@@ -166,9 +196,9 @@ export function is_content_editable(message, edit_limit_seconds_buffer = 0) {
     return false;
 }
 
-export function is_message_sent_by_my_bot(message) {
+export function is_message_sent_by_my_bot(message: Message): boolean {
     const user = people.get_by_user_id(message.sender_id);
-    if (user.bot_owner_id === undefined || user.bot_owner_id === null) {
+    if (!user.is_bot || user.bot_owner_id === undefined || user.bot_owner_id === null) {
```

Comment:
```suggestion
    if (!user.is_bot || user.bot_owner_id === null) {
```

Based on the schema, if `is_bot` is `true`, then `bot_owner_id` isn’t `undefined`.

https://github.com/zulip/zulip/blob/52a9846cdf4abfbe937a94559690d508e95f4065/web/src/state_data.ts#L102-L112

Reply:
Good call, thanks. Made that change.